src/components/PersonConnection/Person.Connection.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link, useNavigate } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+type PersonLinkProps = {
+  person: Person;
+};
+
+export const PersonConnection = ({ person }: PersonLinkProps) => {
+  const { slug, name, sex } = person;
+  const navigate = useNavigate();
+
+  return (
+    <Link
+      to={`/people/${slug}`}
+      className={classNames({ 'has-text-danger': sex === 'f' })}
+      onClick={() => navigate(`/people/${slug}`)}
```

Comment:
The `onClick` handler uses `navigate` to go to the same URL as the `to` prop of the `Link` component. This is redundant because the `Link` component already handles navigation. You can remove the `onClick` handler to simplify the code.